template_engines/backends/docx.py

```diff
@@ -60,3 +60,7 @@ class DocxEngine(ZipAbstractEngine):
     app_dirname = getattr(settings, 'DOCX_ENGINE_APP_DIRNAME', 'templates')
     template_class = DocxTemplate
     zip_root_file = 'word/document.xml'
+
+    def __init__(self, params):
+        params['OPTIONS'] = {'builtins': ['template_engines.templatetags.docx_tags']}
```

Comment:
need to use a dict setdefaults to prevent hard overriding of project configuration

Reply:
or directly use default_builtins in Engine class